.scripts/linux/install-flatbuffers.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+git clone https://github.com/google/flatbuffers.git flatbuffers
+cd ./flatbuffers
+# 1.10.0 release commit
+# git checkout 925c1d77fcc72636924c3c13428a34180c30f96f
+mkdir build
+cd build
+cmake .. -DFLATBUFFERS_BUILD_TESTS=Off -DFLATBUFFERS_INSTALL=On -DCMAKE_BUILD_TYPE=Release -DFLATBUFFERS_BUILD_FLATHASH=Off
+make -j `nproc`
+echo ~
+make DESTDIR=~ install
```

Comment:
How does this differ compared to using `-DCMAKE_INSTALL_PREFIX=/path/to/install` in the cmake command?

Reply:
https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html

`DESTDIR` relocates the whole installation vs pre-pending the path to onto all install directories.

What this actually looks like is:

`CMAKE_INSTALL_PREFIX=~` will install flatbuffers under `/home/<username>/include` and `/home/<username>/bin`
whereas `DESTIR=~` will install flatbuffers under `/home/<username>/usr/local/include` and `/home/<username>/usr/local/bin`

Reply:
Thanks for the explanation, I didn't know that!